styles.css

```diff
@@ -0,0 +1,159 @@
+* {
+  max-width: 1300px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  height: 48px;
+  width: 423px;
+  position: absolute;
+  background-color: #3c3a39;
+  z-index: 1;
+}
+
+ul {
+  list-style: none;
+}
+
+/* Logotype */
+ul li:first-child {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: absolute;
+  width: 94px;
+  height: 44px;
+  left: 5%;
+  top: calc(50% - 44px / 2);
+}
+
+ul li:first-child img {
+  width: 8vh;
+  height: 8vh;
+  border-radius: 5px 5px 0 5px;
+}
+
+/* Icon Menu */
+ul li:last-child {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+ul li:last-child img {
+  width: 24px;
+  height: 24px;
+}
+
+#headline {
+  background: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 390px;
+  top: 48px;
+  background-image: url("./Normal Button/Tertiary/Icons/background-image.png");
+  background-repeat: no-repeat;
+  background-position-x: 100%;
+  background-position-y: 120%;
+}
+
+#headline h1 {
+  width: 327px;
+  height: 124px;
+  font-family: Crete Round, Verdana, Geneva, Tahoma, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  padding-bottom: 10px;
+}
+
+#headline h3 {
+  width: 327px;
+  height: 27px;
+  font-family: Roboto, Verdana, Geneva, Tahoma, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  color: #fff5e1;
+  margin-bottom: 0;
+}
+
+#headline p {
+  width: 327px;
+  height: 123px;
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+#headline .icons {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: space-around;
+  padding: 0;
+  width: 200px;
+  height: 40px;
+  color: #fff;
+}
```

Comment:
- [ ] Good job applying styles to your headline section to match the figma design 🤓. However, please kindly make sure that you use:
- The same type of letter for the `h1` header.
- the same size and position for the social media icons.
as requested by the previous reviewer.

This is the Figma design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/234766578-fec1971c-c2e8-47a4-b729-72d6cf28ac60.png)

This is your design 👇🏽 (almost there 😅)

![design](https://user-images.githubusercontent.com/116701485/234766638-41512ef0-b381-4e1e-bda7-fa5b80f65f7e.png)